config/environments/test.rb

```diff
@@ -54,4 +54,9 @@
   config.active_storage.service = :test
   # Allow SVG's only because we will always serve them in an <img> element
   config.active_storage.content_types_to_serve_as_binary -= ['image/svg+xml']
+
+  # disable PaperTrail for the entire ruby process.
```

Comment:
**question:** Are you sure this is necessary? I think it should be [disabled by default](https://github.com/paper-trail-gem/paper_trail#7b-rspec).